eea/relations/profiles/upgrades/to_121/cssregistry.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0"?>
```

Comment:
I don't think this should be in eea.relations. Maybe EEAPloneAdmin

Reply:
Moved here https://github.com/eea/eea.design/pull/150